src/constants.rs

```diff
@@ -105,16 +105,25 @@ pub static BLACKLISTED_TOKENS: &[&str] = &[
     "8343091f2499fd4b6174a46d067a920a3b851ff9", // jJPY
 ];
 
-pub fn default_decimals() -> BigInt {
-    BigInt::from(18)
+pub fn default_decimals() -> u64 {
+    18
 }
 
-pub fn default_usdc_decimals() -> BigInt {
-    BigInt::from(6)
+pub fn default_decimals_big() -> BigInt {
+    BigInt::from(default_decimals())
 }
 
-pub fn default_usd_price() -> BigDecimal {
-    BigDecimal::from(1000000)
+pub fn default_usdc_decimals() -> u64 {
+    6
+}
+
+// Price per USDC in token's units
+pub fn usdc_smallest_unit_value() -> BigDecimal {
+    BigDecimal::from(10u64.pow(default_usdc_decimals() as u32))
```

Comment:
Just a reminder that BigDecimal and BigInt has built in decimal handlers you might want to check out.

https://docs.rs/substreams/latest/substreams/scalar/struct.BigInt.html#method.to_decimal
https://docs.rs/substreams/latest/substreams/scalar/struct.BigDecimal.html#method.divide_by_decimals

Reply:
Removed this function for now as I don't actually need it, but will use the built in decimal handlers in future.